.travis.yml

```diff
@@ -1,29 +1,44 @@
-language: python
+language: c
+
+os:
+  - osx
+  - linux
+
+compiler:
+  - clang
+  - gcc
 
 env:
-  - TOX_ENV=lint
-  - TOX_ENV=py26
-  - TOX_ENV=py27
-  - TOX_ENV=pypy
-  - TOX_ENV=docs
-  - TOX_ENV=docs-spellcheck
-  - TOX_ENV=docs-linkcheck
+  matrix:
+     - TOX_ENV=lint
+     - TOX_ENV=py26
+     - TOX_ENV=py27
+     - TOX_ENV=pypy
+     - TOX_ENV=docs
+     - TOX_ENV=docs-spellcheck
+     - TOX_ENV=docs-linkcheck
 
 matrix:
   allow_failures:
-    - env: "TOX_ENV=docs-linkcheck"
+    - env: TOX_ENV=docs-linkcheck
+  exclude:
+    - os: osx
```

Comment:
I just shot support an email.  I'm pretty sure this is our mistake, but I just can't see what it is.